pygmt/clib/conversion.py

```diff
@@ -9,6 +9,15 @@
 import numpy as np
 from pygmt.exceptions import GMTInvalidInput
 
+StringArrayTypes = Sequence[str]
```

Comment:
I guess it's time to add the `pygmt/_typing.py` module and define `StringArrayTypes` in it, like what I'm doing in https://github.com/GenericMappingTools/pygmt/pull/3459/files

Reply:
Done at 17c1e9ca28acbb9f35223b0bb770a8dc8bb75843